tests/Doctrine/Tests/ORM/Functional/Ticket/DDC832Test.php

```diff
@@ -41,6 +42,11 @@ public function tearDown(): void
         $sm->dropTable($platform->quoteIdentifier('TREE_INDEX'));
         $sm->dropTable($platform->quoteIdentifier('INDEX'));
         $sm->dropTable($platform->quoteIdentifier('LIKE'));
+
+        if ($platform instanceof PostgreSQLPlatform) {
```

Comment:
The problem with this code is that the test suite ignores the exceptions thrown by the schema tool: https://github.com/doctrine/orm/blob/4f1072e1ac9480acaee37a6fff44f001247392e5/tests/Doctrine/Tests/OrmFunctionalTestCase.php#L348-L356

...and relies on the fact that all tables are created before all sequences. If each sequence is created after its corresponding table, for each test except for the first, the `setUp()` will fail to create all three tables.

We could implement a `dropSchemaForModels()` method similar to `createSchemaForModels()` and use it instead but on the DBAL versions prior to 3.4 it would use `DropSchemaSqlCollector` internally that attempts to drop foreign keys on SQLite even if it doesn't support them. The smallest of all evils, for now, seems to be to work around it.